components/CandidateTestView/CandidateTestView.tsx

```diff
@@ -8,7 +8,7 @@ interface CandidateTestViewProps {
     test: CandidateTestModel;
 }
 const CandidateTestView = (candidateTestViewProps: CandidateTestViewProps): JSX.Element => (
-    <div>
+    <body>
```

Comment:
everything we put inside a next page is automatically inside a body tag I think.

Does this not cause there to be a `body` within a `body`?
If so, we should change it so that this doesn't happen

Reply:
Yes, overzealous landmarking... Removed